tensorflow_datasets/structured/german_credit_numeric.py

```diff
@@ -0,0 +1,86 @@
+# coding=utf-8
+# Copyright 2019 The TensorFlow Datasets Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""German Credit (numeric) dataset."""
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import tensorflow as tf
+import tensorflow_datasets.public_api as tfds
+
+URL = "https://archive.ics.uci.edu/ml/machine-learning-databases/statlog/german/german.data-numeric"
+
+_CITATION = """\
+@misc{Dua:2019 ,
+author = "Dua, Dheeru and Graff, Casey",
+year = "2017",
+title = "{UCI} Machine Learning Repository",
+url = "http://archive.ics.uci.edu/ml",
```

Comment:
This is the link to the whole "repo" - do you have a one to this explicit dataset ?

Reply:
I do, it's https://archive.ics.uci.edu/ml/datasets/Statlog+(German+Credit+Data), however on that page they just tell you to use the blanket citation (https://archive.ics.uci.edu/ml/citation_policy.html).

FWIW, the Iris dataset in TFDS also has the same citation: https://github.com/tensorflow/datasets/blob/master/tensorflow_datasets/structured/iris.py#L32